app/src/main/java/tech/ula/ui/AppListAdapter.kt

```diff
@@ -41,14 +41,12 @@ class AppListAdapter(
             categoriesAndApps.getOrPut(app.category) { arrayListOf() }.add(app)
         }
 
-        val categoriesAndAppsWithDistributionsFirst = categoriesAndApps.toSortedMap(Comparator {
-            first, second ->
-            when {
-                first == topListElement -> -1
-                second == topListElement -> 1
-                else -> 0
-            }
-        })
+        val categoriesAndAppsWithDistributionsFirst = LinkedHashMap<String, List<App>>()
+
+        if (categoriesAndApps.containsKey(topListElement)) {
+            categoriesAndAppsWithDistributionsFirst[topListElement] = categoriesAndApps.remove(topListElement)!!
```

Comment:
I thought it would be more clear that `topListElement` would be at the top of the list rather than needing to know that we wanted distributions at the top. Maybe `firstDisplayCategory` or something would be better?

Reply:
Yeah `firstDisplayCategory` would be a good variable name too 👍